manifests/p/PlayStation/PSPlus/12.1.0/PlayStation.PSPlus.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: PlayStation.PSPlus
+PackageVersion: 12.1.0
+PackageLocale: en-US
+Publisher: Sony Interactive Entertainment Inc.
+PackageName: PlayStation Plus
+License: © 2022 Sony Interactive Entertainment Inc.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
How is [this](https://github.com/microsoft/winget-pkgs/blob/master/manifests/p/PlayStation/PSRemotePlay/5.0.0.02220/PlayStation.PSRemotePlay.locale.en-US.yaml) using copyright as license then?

Reply:
Because sometimes things slip past a reviewer